src/Bicep.Core/Parsing/Lexer.cs

```diff
@@ -90,6 +90,47 @@ public void Lex()
             return segments;
         }
 
+        public static string? TryGetMultilineStringValue(Token stringToken)
+        {
+            const int terminatingQuoteCount = 3;
+            var tokenText = stringToken.Text;
+
+            if (tokenText.Length < terminatingQuoteCount * 2)
+            {
+                return null;
+            }
+
+            for (var i = 0; i < terminatingQuoteCount; i++)
+            {
+                if (tokenText[i] != '\'')
+                {
+                    return null;
+                }
+            }
+
+            for (var i = tokenText.Length - terminatingQuoteCount; i < tokenText.Length; i++)
+            {
+                if (tokenText[i] != '\'')
+                {
+                    return null;
+                }
+            }
+
+            var startOffset = terminatingQuoteCount;
+
+            // we strip a leading \r\n or \n
+            if (tokenText[startOffset] == '\r')
```

Comment:
>tokenText [](start = 16, length = 9)

This matches \r\n | \n, but I believe in the lexer elsewhere we match [\r|\n]+ elsewhere. It's subtly inconsistent, but not an issue for CRLF or LF files. I think it's fine for now, but I think we need to centralize our newline handling in the future. To finish unicode support post-0.3, we will also need to support line break and paragraph break chars.